src/_preflights/forms/form-elements.js

```diff
@@ -13,26 +13,21 @@ export const formElements = `
               box-shadow 0.2s ease;
 }
 
-.input, .textarea, .select {
+.select {
   line-height: 22px;
   background-color: var(--x-form-field-bg);
   border-color: var(--x-form-field-border);
   border-radius: 4px;
   padding: 12px 8px;
 }
-.input:hover, .textarea:hover, .select:hover {
+.select:hover {
   border-color: var(--x-form-field-border-hover);
 }
-.input:focus, .textarea:focus, .select:focus {
+.select:focus {
   border-color: var(--x-form-field-focus);
   outline-color: var(--x-form-field-focus);
 }
 
-.input::placeholder {
```

Comment:
This is needed to be handled in the component classes and components.

Reply:
Wonder if the tw-reset for placeholder will override whatever we set in the components 🤔

Reply:
I see it was added to component-classes here: https://github.com/warp-ds/component-classes/commit/f1cdd745582733ac13a286a2d6ab310e5f01943c
Do you think we should add opacity: 1 at some point, too?

Reply:
@Skadefryd What do you think? Shall we get opacity?